Octokit/Models/Response/Team.cs

```diff
@@ -67,7 +67,7 @@ public Team(string url, int id, string nodeId, string slug, string name, string
         /// <summary>
         /// permission attached to this team
         /// </summary>
-        public StringEnum<Permission> Permission { get; protected set; }
+        public StringEnum<PermissionLevel> Permission { get; protected set; }
```

Comment:
this makes `Value 'pull' is not a valid 'PermissionLevel' enum value` get thrown when trying to serialize a repo object with codeowners that includes a GH team